src/test/java/com/connexta/search/common/SearchManagerImplTest.java

```diff
@@ -202,16 +222,41 @@ void testQueryInvalidEcql() {
   }
 
   @Test
-  void testQueryIdEquals() throws Exception {
+  void testSupportedAttributes() throws Exception {
     // setup
-    final String idQuery = "id = 'value'";
-
-    final URI irmUri = new URI("value");
-    QueryResponse queryResponse = mockQueryResponse(irmUri.toString());
+    final String template = "%s = '%s'";
+    final Map<String, String> queryPairs = new HashMap();
+    queryPairs.put("contents", "lots of words");
+    queryPairs.put("country_code", "USA");
+    queryPairs.put("created", "2019-11-13");
+    queryPairs.put("expiration", "2119-11-01");
+    queryPairs.put("file_url", "http://host/1");
+    queryPairs.put("icid", "floop");
+    queryPairs.put("id", "bloop");
+    queryPairs.put("irm_url", "http://host/2");
+    queryPairs.put("keyword", "key");
+    queryPairs.put("metacard_url", "http://host/3");
+    queryPairs.put("modified", "2019-11-14");
+    queryPairs.put("title", "A Title");
+
+    assertThat(
+        "Expected attributes do not match actual attributes. These collections need to be the same.",
+        List.of(queryPairs.keySet()),
+        containsInAnyOrder(SolrConfiguration.QUERY_TERMS));
+
+    List<String> queryPreds = new ArrayList<>();
+    queryPairs.forEach((k, v) -> queryPreds.add(String.format(template, k, v)));
```

Comment:
❓ What is the purpose of `queryPreds`?

Reply:
Hmmm. Looks like it shouldn't be there anymore now that the entrySet().stream() is working.